contracts/interfaces/IRewardsAggregator.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+// Allows to claim the staking rewards of the Threshold Network.
+interface IRewardsAggregator {
```

Comment:
Why do we need this interface?

Reply:
I don't have a solid argument in favor, and of course, I don't have a strong opinion on this, but I made this because:

1. There was an interface for the previous Merkle distribution contract `CumulativeMerkleDrop`: `contracts/interfaces/ICumulativeMerkleDrop.sol`.
2. It is expected to update this new contract `RewardsAggregator.sol` (see [#TODO](https://github.com/threshold-network/merkle-distribution/blob/aa6f9b075501a9aee24a21c84239ba1253713e65/contracts/RewardsAggregator.sol#L30)) so, having this interface makes more clear what methods we should not modify.

But as I said, there are no solid arguments. Would it be preferable to just remove it?

Reply:
It's fine 👍